niviz_rater/validation.py

```diff
@@ -0,0 +1,51 @@
+import yaml
+import json
+import os
+import yamale
+from yamale.validators import DefaultValidators, Validator
+from bids.config import get_option
+
+SCHEMAFILE = os.path.join(os.path.dirname(__file__), 'data/schema.yaml')
+
+
+def load_json(file):
+    with open(file, 'r') as f:
+        result = json.load(f)
+    return result
+
+
+def get_entities():
+    configfiles = [
+        load_json(file)['entities']
+        for file in get_option('config_paths').values()
+    ]
+    enumstrs = list()
+    for configfile in configfiles:
+        enumstr = [x['name'] for x in configfile]
+        enumstrs.append(enumstr)
+    enumstrs = [item for sublist in enumstrs for item in sublist]
+    return enumstrs
+
+
+class Entities(Validator):
+    tag = 'Entities'
+
+    def _is_valid(self, value):
+        return value in get_entities()
+
+
+def _validate_config(config):
```

Comment:
```suggestion
def validate_config(config):
```
Get rid of the underscore because we wanna expose this method now. In python the "_<NAME>" convention is meant to serve as "hidden" functions.